docs/howto/ci-cd.md

```diff
@@ -0,0 +1,92 @@
+# Suggested CI/CD setup
+
+If your project has more than a single developer, we suggest running `sqlc` as
+part of your CI/CD pipeline. The two commands you'll want to run are `diff` and `vet`
+
+`sqlc diff` ensures that code is up to date. New developers to a project may
+forget to run `sqlc generate`. They also might edit generated code. `diff` will
+catch both scenarios.
+
+```diff
+% sqlc-dev diff
+--- a/postgresql/query.sql.go
++++ b/postgresql/query.sql.go
+@@ -55,7 +55,7 @@
+
+ const listAuthors = `-- name: ListAuthors :many
+ SELECT id, name, bio FROM authors
+-ORDER BY name
++ORDER BY bio
+ `
+```
+
+`sqlc vet` runs a set of lint checks against your SQL queries. These checks are
+helpful in catching anti-patterns before they make it into production. Please
+see the [vet](../reference/cli.html#vet) documentation for a complete guide on adding checks to your
+project.
+
+## General setup
+
+Install `sqlc` using the [suggested instructions](../overview/install).
+
+Create two steps in your pipelines, one for `sqlc diff`and one for `sqlc vet`.
+
+## GitHub Actions
+
+We provide the [setup-sqlc](https://github.com/marketplace/actions/setup-sqlc)
+GitHub Action to install `sqlc`. The action uses the built-in
+[tool-cache](https://github.com/actions/toolkit/blob/main/packages/tool-cache/README.md)
+to speed up the installation process.
+
+The following workflow runs `sqlc diff` on every push.
+
+```yaml
+name: sqlc
+on: [push]
+jobs:
+  diff:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+    - uses: sqlc-dev/setup-sqlc@v3
+      with:
+        sqlc-version: '1.19.0'
+    - run: sqlc diff
+```
+
+We also encourage running [`sqlc vet`](../reference/cli.html#vet). To get the most value out of `vet`,
+you'll want to set up a running database. See the [vet] documentation for a
```

Comment:
[vet] is missing a link url here, but would probably be redundant given the link in the previous sentence so probably just remove square brackets.